test/topology_custom/test_replace_with_encryption.py

```diff
@@ -7,42 +7,54 @@
 from test.pylib.manager_client import ManagerClient
 from test.pylib.scylla_cluster import ReplaceConfig
 
+
```

Comment:
There is some unnecessary noise in this PR:
- a new empty line 10,
- a new space before ")" in lines 50 and 60,
- a bigger indentation in lines below `await manager.servers_add(3,`,
- removing spaces aroung "=".

In general, unrelated stylistic changes should be done in separate commits to not make the important changes harder to review. But this commit is so simple that I can easily understand anyway, so maybe everything is fine. I'm not sure how strict we are about such things.

Reply:
Done!